stl/inc/xutility

```diff
@@ -629,69 +638,56 @@ concept contiguous_iterator = random_access_iterator<_It>
     && requires(const _It& __i) {
         { _STD to_address(__i) } -> same_as<add_pointer_t<iter_reference_t<_It>>>;
     };
+// clang-format on
 
 _EXPORT_STD template <class _Fn, class _It>
-concept indirectly_unary_invocable = indirectly_readable<_It>
-    && copy_constructible<_Fn>
-    && invocable<_Fn&, iter_value_t<_It>&>
-    && invocable<_Fn&, iter_reference_t<_It>>
-    && invocable<_Fn&, iter_common_reference_t<_It>>
-    && common_reference_with<
-        invoke_result_t<_Fn&, iter_value_t<_It>&>,
+concept indirectly_unary_invocable =
+    indirectly_readable<_It> && copy_constructible<_Fn> && invocable<_Fn&, _Indirect_value_t<_It>>
+    && invocable<_Fn&, iter_reference_t<_It>> && invocable<_Fn&, iter_common_reference_t<_It>>
+    && common_reference_with<invoke_result_t<_Fn&, _Indirect_value_t<_It>>,

```

Comment:
No change requested: I am uncomfortable with the drive-by reformatting here. The issue is that the reformatting is mixed with the semantic changes, which makes it harder to see what's going on. I would have preferred for this to be done in a separate PR. That said, it's not *too* complicated and we've already reviewed the changes, so let's leave it as-is.